src/dialog/docs/DialogManager.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export default class DialogManager extends React.PureComponent {
+  static propTypes = {
+    children: PropTypes.func
+  }
+
+  state = {
+    isShown: false,
+    isLoading: false
+  }
+
+  render() {
+    return this.props.children({
+      isShown: this.state.isShown,
+      isLoading: this.state.isLoading,
+      confirmLoading: () => {
```

Comment:
Move all these callback to class methods.

Reply:
Yes.